ml_api/server.py

```diff
@@ -43,13 +43,29 @@ def get_p():
             img = cv2.imdecode(img_array, -1)
             detections = detect(net_main, img, thresh=THRESH)
             return jsonify({'detections': detections})
-        except:
+        except Exception as err:
             sentry_sdk.capture_exception()
+            app.logger.error(f"Failed to get image {request.args} - {err}")
+            abort(
+                make_response(
+                    jsonify(
+                        detections=[],
+                        message=f"Failed to get image {request.args} - {err}",
+                    ),
+                    503,
```

Comment:
Not that it matters that much. But 5xx usually means a server error. Some load balancer/health checker may act accordingly. Let's just use a 400 code here.

Reply:
updated error code to 400.